lizard_auth_server/views.py

```diff
@@ -94,11 +94,31 @@ def portals(self):
         else:
             return self.profile.portals.all()
 
+    @cached_property
+    def oidc_userconsent(self):
+        """
+        Returns UserConsent of the user that is logged in.
+        UserConsent couples foreign key from user and OIDC clients
+        to give consent for logging in on the OIDC client.
+
+        'Client' means a website with an 'OpenID connect' connection with
+        us. Consent means that the user allows the client to use the SSO to
+        log them in.
+        """
+        return UserConsent.objects.filter(user=self.request.user)
+
     @method_decorator(login_required)
     def dispatch(self, request, *args, **kwargs):
         return super(ProfileView, self).dispatch(request, *args, **kwargs)
 
 
+class UserConsentsEditView(TemplateView):
+    """
+    View for listing and *removing* openid connect user consents
+    """
+    template_name = 'lizard_auth_server/userconsents.html'
```

Comment:
Deze template was leeg, geloof ik. Is deze view dus nog nodig?

Reply:
Klopt, done.